packages/flutter_tools/lib/src/vscode/vscode.dart

```diff
@@ -0,0 +1,181 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:convert';
+
+import '../base/common.dart';
+import '../base/file_system.dart';
+import '../base/platform.dart';
+import '../base/version.dart';
+
+// VS Code layout:
+
+// macOS:
```

Comment:
optional: comment blocks related to the filenames might be better placed near _installedMacOS, _installedWindows, and _installedLinux respectively.

Reply:
Done in 38a15aa.